js/data.js

```diff
@@ -6,3 +6,13 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+if (localStorage.getItem('entry-data') !== null) {
+  data = JSON.parse(localStorage.getItem('entry-data'));
```

Comment:
Rather than calling `getItem` twice, I would call it once before the conditional and save the value into a variable that you can use as needed.